f5_openstack_agent/lbaasv2/drivers/bigip/network_helper.py

```diff
@@ -559,11 +617,19 @@ def delete_fdb_entry(
             if len(records) == 0:
                 records = None
 
-        tunnel = bigip.net.fdbs.tunnels.tunnel
-        if tunnel.exists(name=tunnel_name, partition=partition):
-            tunnel.load(name=tunnel_name, partition=partition)
-            tunnel.update(record=records)
+        try:
+            tunnel = bigip.net.fdbs.tunnels.tunnel
```

Comment:
Generally, need to look for resource acquisition code where we do not perform in try-catch.